bundle/manifests/argoproj.io_argocds.yaml

```diff
@@ -1494,9 +1494,17 @@ spec:
                     description: DisableTLSVerification defines whether redis server
                       API should be accessed using strict TLS validation
                     type: boolean
+                  enabled:
+                    description: Enabled is the flag to enable Redis during ArgoCD
+                      installation. (optional, default `true`)
+                    type: boolean
                   image:
                     description: Image is the Redis container image.
                     type: string
+                  remote:
+                    description: Remote specifies the remote URL of the Redis container.
+                      (optionnal, default '')
```

Comment:
Correct typo `optionnal`. Probably you can mention that by default, a local instance managed by the operator is used.